java/src/jmri/util/EarlyInitializationPreferences.java

```diff
@@ -73,10 +79,32 @@ private void load() {
 
         } catch (IOException ex) {
 //            log.warn("Loading startup preferences from {} failed", FILENAME);
-            preferences.setProperty("sun.java2d.uiScale", "1");
-            store();
+            setupNewPreferences();
         }
     }
 
+    private void setupNewPreferences() {
+        int uiScale = 1;
+
+        GraphicsConfiguration gc = GraphicsEnvironment.getLocalGraphicsEnvironment().getDefaultScreenDevice().getDefaultConfiguration();
```

Comment:
This is a long enough chain that I recommend putting it in a try-catch for NullPointerException

Reply:
Good idea. It's fixed now.
Thanks for your comments.